### ISSUE TITLE
servicetitan.321@yahoo.com

### ISSUE THREAD

**Issue by ryanwolcott321** [google/jax#5991]
Please:

- [ ] Check for duplicate issues.
- [ ] Provide a complete example of how to reproduce the bug, wrapped in triple backticks like this:

```python
import jax.numpy as jnp
print(jnp.arange(10))
# [0 1 2 3 4 5 6 7 8 9]
```

- [ ] If applicable, include full error messages/tracebacks.

**Comment by ryanwolcott321** [comment]
Label for down so I can configure factory default settings on iphones and android.

**Comment by jakevdp** [comment]
This looks like the unmodified issue template, so I assume it was opened unintentionally. I'm going to close - please let us know if you have any questions!
